cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/DatasetFramework.java

```diff
@@ -187,6 +187,17 @@ void updateInstance(DatasetId datasetInstanceId, DatasetProperties props)
   Collection<DatasetSpecificationSummary> getInstances(NamespaceId namespaceId) throws DatasetManagementException;
 
   /**
+   * Get all dataset instances in the specified namespace
```

Comment:
Improve javadoc to mention that it returns instances who contains the given set of properties.

Reply:
fixed.